db-migrations/2_test2.sql

```diff
@@ -1,8 +1,8 @@
 CREATE TABLE IF NOT EXISTS game_results (
     id serial PRIMARY KEY,
     game_label VARCHAR(255) NOT NULL,
-    player_num INT,
+    player_num INT NOT NULL,
```

Comment:
Because this migration has already ran, we need to make this change in a new one

Reply:
I think we need the following:
`alter table game_results ALTER COLUMN player_num set NOT NULL;`